applications/machine_tending/skills.py

```
@@ -0,0 +1,140 @@
+#! /usr/bin/env python3
+
+from robogpt_tools.applications.base.skills import *
```

Ruff (0.12.2)

`from robogpt_tools.applications.base.skills import *` used; unable to detect undefined names

(F403)